src/modules/luis.module.ts

```diff
@@ -0,0 +1,52 @@
+import { BotModule } from "../types"
+import { getLog } from "../types/models/chats"
+
+import { User, Message, Client } from "discord.js"
+import pgPromise from "pg-promise"
+import { ParsingResult } from "chrono-node/dist/results"
+
+import MarkovChain from "markovchain"
+const markov = new MarkovChain()
+let vocab: string[] = []
+
+async function initializeMarkov() {
+    try {
+        console.log("Downloading chat data")
+        const logs = await getLog(0) // TODO: support multiple chatbots than hardcoding this
+        const lines = logs.log.replace(/\n/g, " ").split(" ")
+        vocab = lines
+        console.log("Initializing markov data")
+        markov.parse(logs.log)
+        console.log("loaded markov data")
+    } catch (error) {
+        if (error instanceof pgPromise.errors.QueryResultError) {
+            console.error(error)
+        }
+    }
+}
+
+function getResponse(messageContent: string){
+    const messageWords = messageContent.split(" ").filter(word => word !== "!luis")
+    let result = "In Low Priority Queue. Please Ask Me Later"
+    if (vocab.length > 0) {
+        if (Math.random() < (process.env.GIBBERISH_RATIO || 0.5) || messageWords.length == 0) {
```

Comment:
Please use `===` for consistency reasons with the rest of this PR

Reply:
sure